spec/dummy/Gemfile

```diff
@@ -5,3 +5,5 @@ source "https://rubygems.org"
 eval_gemfile File.expand_path("../../Gemfile.development_dependencies", __dir__)
 
 gem "react_on_rails", path: "../.."
+
+gem "turbo-rails"
```

Comment:
please add this gem to the Gemfile.development_dependencies file instead of here.